betelgeuse.py

```diff
@@ -574,6 +621,40 @@ def cli(context, jobs):
     testimony.SETTINGS['minimum_tokens'] = ['id']
 
 
+@cli.command('create-plan')
+@click.option(
+    '--name',
+    default='test-plan-{0}'.format(time.time()),
+    help='Name for new Test Plan.',
+)
+@click.option(
+    '--type',
+    default='release',
+    help='Test Plans can be Releases or Iterations.',
+    type=click.Choice([
+        'release',
+        'iteration',
+    ])
+)
+@click.option(
+    '--parent-name',
+    help='Name of parent Test Plan to link to.',
+)
+@click.option(
+    '--collect-only',
+    help=('Not perform any operation on Polarion, just prints '
+          'collected information.'),
+    is_flag=True,
+)
+@click.argument('project')
+@click.pass_context
+def create_plan(context, name, type, parent_name, collect_only, project):
+    """Create a new test plan in Polarion."""
+    OBJ_CACHE['collect_only'] = collect_only
+    OBJ_CACHE['project'] = project
+    create_test_plan(name, type, parent_name)
```

Comment:
since this is not using multiprocessing the function create_test_plan body can be moved to here. Also the return of that function can be dropped since it is not being used for anything.